crates/types/src/bounding_box.rs

```diff
@@ -0,0 +1,26 @@
+use coordinate_systems::Pixel;
+use geometry::rectangle::Rectangle;
+use serde::{Deserialize, Serialize};
+use serialize_hierarchy::SerializeHierarchy;
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, SerializeHierarchy)]
+pub struct BoundingBox {
+    pub bounding_box: Rectangle<Pixel>,
+    pub score: f32,
+}
+
+impl BoundingBox {
+    pub fn new(score: f32, bounding_box: Rectangle<Pixel>) -> Self {
+        Self {
+            bounding_box,
+            score,
+        }
+    }
```

Comment:
Do you want to drop this method? And use the constructor?

Reply:
Done